code/processes/wdb.q

```diff
@@ -528,21 +642,23 @@ getsortparams:{[]
 
 
 /- make sure to request connections for all the correct types
-.servers.CONNECTIONS:(distinct .servers.CONNECTIONS,.wdb.hdbtypes,.wdb.rdbtypes,.wdb.gatewaytypes,.wdb.tickerplanttypes,.wdb.sorttypes,.wdb.sortworkertypes) except `
+.servers.CONNECTIONS:distinct .wdb.hdbtypes,.wdb.rdbtypes,.wdb.gatewaytypes,.wdb.tickerplanttypes,.wdb.sorttypes,.wdb.sortworkertypes,.wdb.idbtypes;
```

Comment:
why have we removed `.servers.CONNECTIONS` here? the reason for it being there is in case a user has configured additional processes to connect to in config etc., I think we should still have it here

```suggestion
.servers.CONNECTIONS:(distinct .servers.CONNECTIONS,.wdb.hdbtypes,.wdb.rdbtypes,.wdb.gatewaytypes,.wdb.tickerplanttypes,.wdb.sorttypes,.wdb.sortworkertypes,.wdb.idbtypes) except `
```

Reply:
Yes you might be right! Added back.